{"name": "Проверка тело API на ошибку аторизации при неправильном пароле", "status": "broken", "statusDetails": {"message": "KeyError: 'first_name'", "trace": "@pytest.fixture\n    def new_courier():\n        payload = get_register_payload()\n>       return register_new_courier_and_return_login_password(payload)\n\nconftest.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\npayload = {'firstName': 'ingqubsunp', 'login': 'mxudladmyz', 'password': 'bxfgildszt'}\n\n    def register_new_courier_and_return_login_password(payload):\n    \n        login_pass = []\n    \n        response = requests.post(COURIER_URL, data=payload)\n    \n        if response.status_code == 201:\n            login_pass.append(payload['login'])\n            login_pass.append(payload['password'])\n>           login_pass.append(payload['first_name'])\nE           KeyError: 'first_name'\n\nhelpers.py:38: KeyError"}, "start": 1716814203687, "stop": 1716814203687, "uuid": "f0e305c9-b391-42f8-a4cb-cf8a64effabc", "historyId": "b0fb7c981c96245c0fe0bd4d68a4ea3d", "testCaseId": "b0fb7c981c96245c0fe0bd4d68a4ea3d", "fullName": "tests.test_login.TestLogin#test_wrong_login_password_body", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-8BB0DG15"}, {"name": "thread", "value": "17604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}